File: app/src/main/java/zo/den/testtask4/data/network/pojo/Rss.java
package zo.den.testtask4.data.network.pojo;


import org.simpleframework.xml.Attribute;
import org.simpleframework.xml.Element;
import org.simpleframework.xml.Root;

@Root(name = "rss")
public class Rss {

    @Attribute(name = "version")
    String version;

    @Element(name = "channel")
    Channel channel;


    public Channel getChannel() {
        return channel;
    }

    @Override
    public String toString() {
        return "RSS{" +
                "version='" + version + '\'' +
                ", channel=" + channel +
                '}';
    }
}

File: app/src/main/java/zo/den/testtask4/data/network/pojo/Rss.kt
package zo.den.testtask4.data.network.pojo


import org.simpleframework.xml.Attribute
import org.simpleframework.xml.Element
import org.simpleframework.xml.Root

@Root(name = "rss", strict = false)
class Rss  constructor(){

    @set:Attribute(name = "version", required = false)
    @get:Attribute(name = "version", required = false)
    var version: String? = null

    @set:Element(name = "channel", required = false)
    @get:Element(name = "channel", required = false)
    var channel: Channel? = null

    override fun toString(): String {
        return "RSS{" +
                "version='" + version + '\''.toString() +
                ", channel=" + channel +
                '}'.toString()
    }
}